examples-experimental/potree/src/pointcloud-tileset.ts

```diff
@@ -45,7 +58,7 @@ export class PointcloudTileset {
   }
 
   get tiles() {
-    return [];
+    return this._selectedNodes;
```

Comment:
Nit: I would prefer that we don't overuse getters. There should be a good reason for using a getter. I.e. 
- Why use a getter here?
- Why not just have a method `getTiles()`?

Reply:
`tileset.tiles` is used by deck.gl